packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.ts

```diff
@@ -117,14 +119,8 @@ function doMoveLine(
             });
 
             const newLine: PairOfPoints = [
-                snap({
-                    snapStep,
-                    point: vec.add(currentLine[0], change),
-                }),
-                snap({
-                    snapStep,
-                    point: vec.add(currentLine[1], change),
-                }),
+                snap(snapStep, vec.add(currentLine[0], change)),
+                snap(snapStep, vec.add(currentLine[1], change)),
```

Comment:
Suggestion: I'm not sure I want to lose the named arguments on `snap`. Since the point and snap step params have the same structure (a pair of numbers) it's too easy to mix them up.

Maybe a compromise like this would work?

```ts
snap(point, {snapStep: [0.5, 0.5]})
```

This makes it pretty clear which argument is which, and it has the additional advantage that we can pass the whole graph state as the second argument:

```ts
snap(point, state)
```

Reply:
I'll revert the change. I just really dislike the "all arguments as an arg object" pattern; I think it's needless complexity and we're adding that complexity to solve a problem that IDEs fix:

<img width="580" alt="Screenshot 2024-04-04 at 2 02 27 PM" src="https://github.com/Khan/perseus/assets/16308368/bcff3186-dc8b-42e6-9c20-ca4508ecc6f1">

I also think this makes the TS signature more useful.

<img width="589" alt="Screenshot 2024-04-04 at 2 05 53 PM" src="https://github.com/Khan/perseus/assets/16308368/a7cc3cc2-44d3-49f2-9cac-a007273a0a32">

<img width="530" alt="Screenshot 2024-04-04 at 2 04 55 PM" src="https://github.com/Khan/perseus/assets/16308368/4a7e3980-1d03-4949-aea9-a884f1c0e12a">

But I figured I'd get push back.